source/messaging/extending-messaging-with-integrations.md

```diff
@@ -1,9 +1,111 @@
 ## Integrations
 
-Settings for integrations are accessible from the **Main Menu** by clicking the three dots at the top of the channels pane. Clicking **Integrations** opens a page where you can view and configure incoming webhooks, outgoing webhooks, and slash commands for your team. If you can't see an **Integrations** option, then your System Admin may have only given Admins access.
+Settings for integrations are accessible from the **Main Menu**. Select **Integrations** to open a page where you can view and configure incoming webhooks, outgoing webhooks, and slash commands for your team. If you can't see an **Integrations** option, then your System Admin may have only given Admins access.
 
 [Visit our app directory](https://about.mattermost.com/default-app-directory/) for dozens of open source integrations to common tools like Jira, Jenkins, GitLab, Trac, Redmine, and Bitbucket, along with interactive bot applications (Hubot, mattermost-bot), and other communication tools (Email, IRC, XMPP, Threema) that are freely available for use and customization.
 
+### Microsoft integrations
+
+There are several ways to connect your Microsoft tools to Mattermost. Take a look at the tools recommended below, or visit the Marketplace to find others.
+
+#### Office 365 Calendar (E20)
+
+- Two-way integration between Mattermost and Office 365 Calendar, developed by Mattermost.
+- Receive a daily summary of calendar events, and accept or decline new events.
+- Reflect user status as "Do Not Disturb" when in a meeting scheduled via Outlook Calendar.
+- Docs: https://mattermost.gitbook.io/plugin-mscalendar
+
+#### Skype for Business
+
+- Start and join voice calls, video calls, and use screen-sharing with your team members. Developed by kosgrz and maintained by Mattermost.
```

Comment:
It looks like we're using the developer's GitHub handle? If so, does it make sense to link to his GitHub account as well?

Reply:
In this case we do mention them in the README but as it's maintained by us we don't link to their account directly in our docs.

Reply:
Makes sense. Please disregard the remaining comments of the same suggestion. :)